src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></h4>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :aria-label="selectLabel"
+        :class="ulClass"
+        role="listbox"
+        :id="id"
+      >
+        <li 
+          :class="liClass" 
+          @click="openList"
+          role="option"
+          :style="selectedOption ? {color: '#000000'} : {color: '#cccccc'}"
+          >{{ selectedOption ? selectedOption.text : placeholder }}
+          <span class="dropdown-arrow"><i class="fa fa-caret-down" aria-hidden="true" style="color: #000000"></i></span>
+        </li>
+        <li
+          :class="liClass"
+          v-if=opened
+          v-for="option in options"
+          :key="option.id"
+          :value=option.id
+          @click="selectOption(option)"
+          role="option"
+        > {{ option.text }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+
+const options = []
+
+export default {
+  name: 'b-select',
+  props: {
+
+    id: {
+      type: String,
+      required: true
+    },
+    selectOptions: {
+      type: Array
+    },
+    isRequired: {
+      type: Boolean,
+      required: true
+    },
+    selectLabel: {
+      type: String,
+      required: true
+    },
+    preSelectedOption: {
```

Comment:
I like `defaultSelection`

Reply:
Just realized that Steven deleted his comment about this, but I like `default` over `preSelected`

Reply:
its not a default though, if someone was to built an "edit" capability, it would be preselected?

Reply:
What do you mean "build an edit capability"?

Reply:
my english is good.  I mean like if you go into an item that was already made and need to preload previously selected options, its not a default at that point.

Reply:
I am definitely open to calling it something else, i just don't think 'default' is necessarily accurate?

Reply:
Aaaaah. I got you. Ok, fair point.